app/js/lib/server/network.js

```diff
@@ -66,8 +66,16 @@ export default class Network extends EventDispatcher {
    * @return {Promise}
    */
   fetchJSON(url, method = 'GET', body = undefined) {
-    return this[p.fetch](url, 'application/json', method, body)
-      .then((response) => response.json());
+    const jsonMime = 'application/json';
+    return this[p.fetch](url, jsonMime, method, body)
+      .then((response) => {
+        const contentType = response.headers.get('Content-Type');
+        if (!contentType.startsWith(jsonMime) && response.ok) {
```

Comment:
because the content type is like `application/json; charset=utf-8` so `startsWith` is the right tool here :)